core/src/main/java/gov/nasa/jpl/mudrod/driver/ESDriver.java

```diff
@@ -78,7 +81,6 @@
   private transient Client client = null;
   private transient Node node = null;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "node" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)